_includes/nav_v1.0.html

```diff
@@ -8,13 +8,8 @@
 					GETTING STARTED
 					 -->
 	<li class="navContainer">Getting Started
```

Comment:
Just to confirm this is what you intended-- it used to be a sibling, but now it's a child of the "Getting Started" li. Is that right?